examples/runners/wgpu/src/lib.rs

```diff
@@ -0,0 +1,204 @@
+use winit::{
+    event::{Event, KeyboardInput, VirtualKeyCode, WindowEvent},
+    event_loop::{ControlFlow, EventLoop},
+    window::Window,
+};
+
+async fn run(event_loop: EventLoop<()>, window: Window, swapchain_format: wgpu::TextureFormat) {
+    let size = window.inner_size();
+    let instance = wgpu::Instance::new(wgpu::BackendBit::PRIMARY);
+
+    // Wait for Resumed event on Android; the surface is only needed early to
+    // find an adapter that can render to this surface.
+    let mut surface = if !cfg!(target_os = "android") {
```

Comment:
nit: `!cfg!` is a little gross, could we swap the true/false branches if this statement?

i.e. `if cfg!(..) { None } else { Some(..) }`

Reply:
Yeah I noticed it looks weird this way. I'm a fan of keeping `else { None };` as the last block but that doesn't really work out here, unless using `cfg!(not(target_os = "android"))`. That's more characters as well so I'll flip them around :+1: